demo/dsa-accel-config-demo/Dockerfile

```
@@ -1,9 +1,10 @@
-FROM fedora:32 AS builder
+FROM fedora:35 AS builder
 
-RUN dnf install -y wget autoconf automake libtool m4 diffutils file make dnf-utils
-RUN dnf install -y libuuid-devel json-c-devel kmod-devel libudev-devel
+RUN dnf install -y \
+    wget autoconf automake libtool m4 diffutils file make dnf-utils patch \
+    libuuid-devel json-c-devel kmod-devel libudev-devel
```

was it so that `kmod` and `udev` where not needed?
```suggestion
    libuuid-devel json-c-devel
```

---

Yes, both not needed, dropped them